package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@lob/ui-components",
-  "version": "1.0.0-beta.17",
+  "version": "1.0.0-beta.19",
```

Comment:
How come 18 is being skipped here?

Reply:
The order of things going in, it wasn't actually skipped - i had another PR that was about to be merged first that was .18